### ISSUE TITLE
Error message when finding cluster markers

### ISSUE THREAD

**Issue by JoshuaCumming** [satijalab/seurat#847]
I am receiving the following when trying to find cluster markers:
```R
> cluster1.markers <- FindMarkers(object = DNUC, ident.1 = 1, min.pct = 0.25)
   |                                                  | 0 % ~calculating  Error in model.frame.default(formula = countdata.test[x, ] ~ coldata[,  : 
  invalid type (list) for variable 'countdata.test[x, ]'
```
Any ideas how to fix?

**Comment by JoshuaCumming** [comment]
Hi,

I have found that I don't receive this error if I have used the normalize function in the analysis. However, I have already normalized my data so was hoping to avoid the normalization step. Instead I just put my data in a log scale using: "log(data+1)" and it was working fine up until the findmarkers function. Does anyone know whether I can fix the above problem first? Also what is the danger of normalizing already normalized data?

Thanks,

Joshua
